sphinxcontrib/spelling/builder.py

```diff
@@ -210,14 +210,19 @@ def _find_misspellings(self, docname, doctree):
                 # Check the text of the node.
                 misspellings = self.checker.check(node.astext())
                 for word, suggestions, context_line in misspellings:
-                    msg_parts = [docname + '.rst']
-                    if lineno:
-                        msg_parts.append(darkgreen(str(lineno)))
-                    msg_parts.append(red(word))
+                    msg_parts = [red(word)]
                     msg_parts.append(self.format_suggestions(suggestions))
                     msg_parts.append(context_line)
-                    msg = ':'.join(msg_parts)
-                    logger.info(msg)
+                    if self.config.spelling_warning:
```

Comment:
It will be less surprising to users if we have a consistent log format and only change the level of the log message based on the option. Does logger.info() accept the location argument?

Reply:
Good point.  I updated the PR. It makes the code simpler, too. 

Here is an example with warning and -q. badlink is there to show what other warnings looks like:

> rscohn2@rscohn1-MOBL:test$ sphinx-build -M spelling "source" "build"  -q
/home/rscohn2/local/Projects/doc/test/source/index.rst:11: WARNING: Unknown target name: "badlink".
/home/rscohn2/local/Projects/doc/test/source/index.rst:9: WARNING: Spell check: sntence: Misspelling in a sntence.
WARNING: Found 1 misspelled words
rscohn2@rscohn1-MOBL:test$

Here is info and no -q:

> rscohn2@rscohn1-MOBL:test$ sphinx-build -M spelling "source" "build"
Running Sphinx v3.3.0
Initializing Spelling Checker 7.0.3.dev2
making output directory... done
Ignoring wiki words
Ignoring acronyms
Ignoring Python builtins
Ignoring importable module names
Ignoring contributor names
Looking for custom word list in /home/rscohn2/local/Projects/doc/test/source/spelling_wordlist.txt
Scanning contributors
building [mo]: targets for 0 po files that are out of date
building [spelling]: all documents
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index
>
> /home/rscohn2/local/Projects/doc/test/source/index.rst:11: WARNING: Unknown target name: "badlink".
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index
/home/rscohn2/local/Projects/doc/test/source/index.rst:9: Spell check: sntence: Misspelling in a sntence.
Writing /home/rscohn2/local/Projects/doc/test/build/spelling/index.spelling
>
> WARNING: Found 1 misspelled words
build succeeded, 2 warnings.
rscohn2@rscohn1-MOBL:test$